en/debug/eclipse_jlink.md

```diff
@@ -0,0 +1,83 @@
+# MCU Eclipse/J-Link Debugging for PX4
+
+This topic explains how to setup and use [MCU Eclipse](https://gnu-mcu-eclipse.github.io/) with a *Segger Jlink adapter* to debug PX4 based-devices.
+
+
+## Required Hardware
+
+- [J-Link EDU Mini](https://www.segger.com/products/debug-probes/j-link/models/j-link-edu-mini/)
+- Pixhawk Debug Adapter Board
```

Comment:
Where do we get this. HOw is it attached?

Reply:
This debug adapter board is currently only available internally. So we may be just call it an adapter to connect segger jlink to debug port.